docs/rendering-spec.md

```diff
@@ -0,0 +1,524 @@
+| numberSuffix              | string  | Applicable for channels with numeric fields only.                                                                                             |
+| numberThousandsSeparators | boolean | Applicable for channels with numeric fields only.                                                                                             |
+| numberPrefix              | string  | Applicable for channels with numeric fields only.                                                                                             |
+| plotOnSecondaryAxis       | boolean | Applicable for the following chart types only: Continuous Area, Grouped Column, Grouped Combo, Stacked Combo, Continuous Line, Discrete Line. |
+| wrapText                  | boolean | Applicable for Table charts only.                                                                                                             |
+
+## Customizable Axes Options
+
+All axes customizations are optional. For a more detailed description of the axes customizations please check our [MongoDB Charts documentation for customizable axes](https://www.mongodb.com/docs/charts/customize-charts/value-axis-options/).
+
+| Customization | Type                                    | Applicability                                                                                                                                                                                                                                                                                                                                                 |
+| ------------- | --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| labelOverride | string                                  | Overriding the label of the axis.                                                                                                                                                                                                                                                                                                                             |
+| labelAngle    | 'vertial' \| 'horizontal' \| 'diagonal' | Applicable for the following chart types only: All Column charts, 100% Stacked Area, Discrete Area, Discrete Line, Grouped Combo, Stacked Combo, Heatmap.                                                                                                                                                                                                     |
```

Comment:
```suggestion
| labelAngle    | 'vertical' \| 'horizontal' \| 'diagonal' | Applicable for the following chart types only: All Column charts, 100% Stacked Area, Discrete Area, Discrete Line, Grouped Combo, Stacked Combo, Heatmap.                                                                                                                                                                                                     |
```

Reply:
fixed all the typos and format issues here: https://github.com/mongodb-js/charts-embed-sdk/pull/86/commits/731549b69f634969e5f2a52feb0e501cc8ba132a